src/main/java/com/rusticisoftware/tincan/LRS.java

```diff
@@ -29,11 +29,12 @@
 
     StatementLRSResponse saveStatement(Statement statement);
     StatementsResultLRSResponse saveStatements(List<Statement> statements);
-    StatementLRSResponse retrieveStatement(String id);
-    StatementLRSResponse retrieveVoidedStatement(String id);
+    StatementLRSResponse retrieveStatement(String id, boolean attachments);
+    StatementLRSResponse retrieveVoidedStatement(String id, boolean attachments);
```

Comment:
This is changing the public interface and would necessitate a major version bump. I think it'd be better to add the method signatures and leave the existing ones in place for backwards compatibility. Presumably the code change is simple, with `false` being the default.